EKF/covariance.cpp

```diff
@@ -867,16 +860,35 @@ void Ekf::fixCovarianceErrors(bool force_symmetry)
 
 void Ekf::resetMagRelatedCovariances()
 {
-	// set the quaternion covariance terms to zero
+	resetQuatCov();
+	resetMagCov();
+}
+
+void Ekf::resetQuatCov(){
```

Comment:
This seems odd. I expect a function resetQuatCov to do more then just zeroing it. But that is what was done earlier. Maybe we should reset it to the initial value?

Reply:
Maybe, yes. This is the reason I created this intermediate function, but since this PR is mostly "non-functional" I didn't want to change it here.

Reply:
Actually I changed it here: https://github.com/PX4/ecl/pull/693/commits/5c6a135e2f53d602840a6354ea096561feb3a1ea
Using the initial uncertainty makes more sense than zero